en/index.md

```diff
@@ -407,6 +407,50 @@ rather than plain English. Try to keep it simple.
 #21 No double space after a period/full stop
 -->
 
+## Admonitions
+
+Admonitions (also referred to as "admonishments", "callouts" or
+"notifications") are a device used in documentation to draw attention to a
+particular statement or paragraph of text. Typically their use is to highlight:
+
+ - A consequence of performing a particular action
+ - An additional source of information which is useful but not required
+ - A helpful tip that will save effort/time
+ - A reminder of some pre-requisite or restriction
+
+The [Vanilla framework][], which is used to publish most of our documentation,
+only allows for a particular set of pre-defined admonitions (or "Notifications"
+as they are referred to in that framework): **Information** and **Warning**.
+
+Although other systems may allow for more scope it is advised to **only use these 
+two types**, in the circumstances indicated below.
+
+### Warning
+
+The **Warning** style of admonition is appropriate where certain actions or
+lack of action could have negative consequences - for example the loss of data
+or an interruption to a service.
```

Comment:
The "interruption to a service" sounds strange to me.

What about this?

```suggestion
or an interruption of a service.
```

Or this

```suggestion
or a service interruption.
```

I could be wrong of course

Reply:
It would normally be "to a service" or "of service" but if you find it sounds funny then maybe other people also would,  so I'm happy to go with 'service interruption'